app/presenters/outcome_presenter.rb

```diff
@@ -1,7 +1,25 @@
 require 'erubis'
 
 class OutcomePresenter < NodePresenter
-  class OutcomeTemplateMissing < StandardError; end
+  class ViewContext
+    def initialize(state)
+      @state = state
+      state_methods = state.methods - SmartAnswer::State.new(nil).methods
+      @state_reader_methods = state_methods.reject { |m| m =~ /=$/ }
+    end
+
+    def method_missing(method, *args, &block)
```

Comment:
Oh right. I was thinking we could call `self.class.delegate *state_reader_methods, to: :state`, but I overlooked that it depends on the instance protocol rather than the class.

I guess we'll have to stick to method_missing, but I would suggest overriding [`respond_to?`](https://robots.thoughtbot.com/always-define-respond-to-missing-when-overriding) as well.